src/collections/_documentation/data-management/rollups.md

```diff
@@ -45,29 +72,17 @@ If a template is involved the logic is similar to grouping by stack trace but ob
 
 If everything else fails, grouping falls back to messages. Hereby ideally the grouping only uses the message without the parameters, but if that is not available, it uses the message attribute.
 
-## Customize Grouping with Fingerprints {#custom-grouping}
-
-For some very advanced use cases you can override the Sentry default grouping using the `fingerprint` attribute. In supported SDKs, this attribute can be passed with the event information, and should be an array of strings. 
-
-If you wish to append information, thus making the grouping slightly less aggressive, you can do that as well by adding the special string `{% raw %}{{ default }}{% endraw %}` as one of the items (note that the double braces are important).
-
-### Minimal example
-
-This minimal example will put all exceptions of the current scope into the same issue/group:
-
-{% include components/platform_content.html content_dir='set-fingerprint' %}
-
-There are two common real-world use cases for the `fingerprint` attribute:
+## Use Cases
 
-### Example: Split up a group into more groups (groups are too big)
+### Create more granular groups for generic errors
```

Comment:
I think the combination of this heading and the one after would be easier to parse - the goal being that if I glance at them both, it's immediately obvious to me which one I want - if they had parallel construction. So I actually like (some version of) the old version better. Something like

**Split one group into multiple groups**

and

**Merge multiple groups into one group**?

Or, if you prefer the new first-heading language better, maybe

**Group errors more granularly**

and

**Group errors more aggressively**?